README.md

```diff
@@ -47,6 +47,54 @@ Whether you are building healthcare applications, integrating with AI assistants
     cp .env.example .env
     ```
 
+## Development & Testing
+
+### Installing Development Dependencies
+
+To run tests and contribute to development, install the test dependencies:
+
+**Using pip:**
+```bash
+# Install project in development mode with test dependencies
+pip install -e .[test]
+
+# Or install from requirements file
+pip install -r requirements-dev.txt
+```
+
+**Using uv:**
+```bash
+# Install development dependencies
+uv sync --dev
+```
+
+### Running Tests
+
+The project includes a comprehensive test suite with 123+ tests covering all major functionality:
+
+```bash
+# Simple test runner
+python run_tests.py
+
+# Or direct pytest usage
+PYTHONPATH=src python -m pytest tests/ -v --cov=src/fhir_mcp_server
+```
+
+**Test Features:**
+- 🧪 **123 tests total** with comprehensive coverage
```

Comment:
Do we need to mention the exact number? can't we say more than 100 tests or something similar?